datafusion-examples/examples/csv_sql.rs

```diff
@@ -48,5 +49,22 @@ async fn main() -> Result<()> {
     // print the results
     df.show().await?;
 
+    // query with
```

Comment:
expand on this comment a bit? seems it was cutoff, so like `query compressed CSV with specific options` etc.

Reply:
fixed.